evasquez26/angular4-startup
# Webpack configuration for production environment on digital garage

## Story / Task

[ONE-139](https://bixlabs.atlassian.net/browse/ONE-139)


## What this PR does?

Sets the configuration for a deploy to digital garage


### Before 

We did not have a proper configuration for a deployment to digital garage

### After

We have a correct configuration for a deployment to digital garage

### How to test / reproduce

- Execute `npm run start`
- Navigate to http://localhost:8080
- It should show the application running in production mode

### TODOs
N/A

### Related PRs
N/A

## Migrations

NO
